profiling/src/config.rs

```diff
@@ -220,6 +228,20 @@ pub(crate) unsafe fn profiling_experimental_timeline_enabled() -> bool {
     get_bool(ProfilingExperimentalTimelineEnabled, false)
 }
 
+/// # Safety
+/// This function must only be called after config has been initialized in
+/// rinit, and before it is uninitialized in mshutdown.
+pub(crate) unsafe fn profiling_experimental_exception_enabled() -> bool {
+    get_bool(ProfilingExperimentalExceptionEnabled, false)
+}
+
+/// # Safety
+/// This function must only be called after config has been initialized in
+/// rinit, and before it is uninitialized in mshutdown.
+pub(crate) unsafe fn profiling_experimental_exception_sampling_distance() -> i64 {
```

Comment:
Nope, thinking more about this, it should be unsigned and `> 0`, so I'll add some checks. I do not think that there is a situation where you'll set the sampling distance to something that might not fit into a `u16`, but this might also be too limiting OTOH.
I'll add some checks and we can go with `u32` WDYT?